tools/functional-tester/etcd-tester/tester.go

```diff
@@ -227,7 +229,12 @@ func (tt *tester) cleanup() error {
 func (tt *tester) cancelStressers() {
 	plog.Printf("%s canceling the stressers...", tt.logPrefix())
 	for _, s := range tt.cluster.Stressers {
-		s.Cancel()
+		switch s.(type) {
+		case *leaseStresser: // lease stresser doesn't need to be cancelled after each fault injection
```

Comment:
This is bad; all stressers should follow the same path or else it defeats the purpose of having an interface. Type assertions are really a last resort.